back/integration-tests/run.sh

```diff
@@ -10,13 +10,15 @@ export MSYS_NO_PATHCONV=1 # needed for windows
 EXIT_CODE=0
 
 startcontainers(){
```

Comment:
il faudrait modifier le texte d'aide pour la commande `-u spin up containers and run prisma deploy`